amethyst/src/main/java/com/vitorpamplona/amethyst/ui/actions/NewPostView.kt

```diff
@@ -1845,6 +1847,38 @@ fun ImageVideoDescription(
                 )
             }
 
+            Row(
+                verticalAlignment = Alignment.Bottom,
+                horizontalArrangement = Arrangement.Center,
+                modifier = Modifier.fillMaxWidth(),
+            ) {
+                Text("Media Quality")
+            }
+
+            Row(
+                verticalAlignment = Alignment.CenterVertically,
+                modifier = Modifier.fillMaxWidth(),
+            ) {
+                Column {
+                    Slider(
+                        value = sliderPosition,
+                        onValueChange = { sliderPosition = it },
+                        valueRange = 0f..2f,
+                        steps = 1,
+                    )
+
+                    Text(
+                        text =
+                            when (sliderPosition.toInt()) {
+                                0 -> CompressorQuality.LOW.toString()
+                                1 -> CompressorQuality.MEDIUM.toString()
+                                2 -> CompressorQuality.HIGH.toString()
+                                else -> CompressorQuality.MEDIUM.toString()
+                            },
+                    )
+                }
+            }
+
```

Comment:
I wonder if in addition to the UX above, we should have a global media compression setting for the user, but which can be overridden here. cc @vitorpamplona

Reply:
I'm considering that for version 2 but it's always a balance of not cluttering settings screen versus convenience.

Currently the default is set to medium in UI and most users will probably just click upload using medium.

A global setting would be useful for users that always want to upload low or high.

Reply:
> I'm considering that for version 2 but it's always a balance of not cluttering settings screen versus convenience.
> 
> Currently the default is set to medium in UI and most users will probably just click upload using medium.
> 
> A global setting would be useful for users that always want to upload low or high.

Exactly.